source/features/linkify-user-location.tsx

```diff
@@ -41,7 +42,8 @@ void features.add(import.meta.url, {
 
 Test URLs
 
-https://github.com/docubot
-https://github.com/
+- isUserProfile: https://github.com/docubot
```

Comment:
Should probably be replaced with a real user using a real location. I'd go for: https://github.com/mysticatea